frontend/src/components/Blog/AnimatedMessage.tsx

```diff
@@ -8,8 +8,8 @@ const messages = [
   "🎉 You've reached the end of the feed, but not the end of your journey! 🎉",
   "👏 You've successfully surfed the infinite wave of content. Time to ride the next one! 👏",
   "🚀 You've made it to the center of the content universe. Now, let's explore new galaxies! 🚀",
-  "🌟 No more content to load, but infinite possibilities await! Keep chasing your dreams..🌟",
-  "🎁 Bravo! You've reached the depths of the feed. Now, let's dive into something new! 🎁"
+  '🌟 No more content to load, but infinite possibilities await! Keep chasing your dreams..🌟',
```

Comment:
Use of inconsistent quotes. This should've been fixed using lint commands.
Did you run lint:fix and format commands ?

Reply:
After i ran npm run lint-fix and npm run format, these changes were there.